src/classes/HH_Container_LCTRL.cls

```diff
@@ -271,13 +272,24 @@ public with sharing class HH_Container_LCTRL {
             'MailingCountry', 
             'MailingLatitude', 
             'MailingLongitude'};
-        if (UTIL_Describe.isMiddleNameEnabled()) {
-            setContactFields.add('Middlename');
-        }
+
+        addContactMiddleNameField(setContactFields);
+
         return setContactFields;
     }
 
     /*******************************************************************************************************
+    * @description Adds MiddleName field into the set of field names
+    * @param fieldNames Set of field names 
+    * @return void
+    */
+    private static void addContactMiddleNameField(Set<String> fieldNames) {        
+        if (UTIL_Describe.isMiddleNameEnabled()) {
+            fieldNames.add('MiddleName');
```

Comment:
I see your point. It does fail with `Lastname` instead of `LastName`. Certainly `MiddleName` is more correct as a field name than what I had as `Middlename`. I wonder where that existing backlog item is, or whether it's even a priority given that most Admin's will choose a pre-defined naming convention, and if they choose to override it'll default to one with the proper case. Maybe we should update the documentation to state that field names are case sensitive. What do you think @smarovich?